src/QTRunAction.cc

```diff
@@ -18,7 +18,6 @@ EGRunAction::EGRunAction(QTEventAction* eventAction, G4String name, G4int na)
   auto analysisManager = G4AnalysisManager::Instance();
 
   // Create directories
-  analysisManager->SetDefaultFileType("root");
```

Comment:
I believe this is no longer necessary in v11.X - file types are inferred from extensions. Tagging @TomGoffrey @yramachers

Reply:
Ok, that was copy/paste from older versions. Would be good if that can disappear.